packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1060,7 +1056,9 @@ new PrismaClient({
      * @param middlewareIndex
      * @returns
      */
-    private _request(internalParams: InternalRequestParams): Promise<any> {
+    async _request(internalParams: InternalRequestParams): Promise<any> {
+      const tracer = trace.getTracer('prisma')
```

Comment:
Register an otel tracer for prisma.

Reply:
Should this always happen or only if Otel is detected/present?

Reply:
Yes, otel is designed to do no-ops if it has not been set up.